iroh-resolver/src/chunker.rs

```diff
@@ -73,6 +73,12 @@ impl Chunker {
             }
         }
     }
+
+    pub fn chunk_size(&self) -> usize {
```

Comment:
most chunkerd will not have this, should avoid an explicit assumption in the api that this exists.